main.tf

```diff
@@ -1,20 +1,71 @@
-provider "aws" {
```

Comment:
I don't think you wanted to delete this one.

Reply:
re-added 👍